### PR TITLE
add Harvester provider

### PR DESCRIPTION
**Pull request by futuretea**
**Related issues**
harvester/harvester#943

### CODE REVIEW

File: cmd/provider_cmd_harvester.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+			provider := newHarvesterProvider()
+			err := Import(provider, options, []string{})
+			if err != nil {
+				return err
+			}
+			return nil
+		},
+	}
+	cmd.AddCommand(listCmd(newHarvesterProvider()))
+	baseProviderFlags(cmd.PersistentFlags(), &options, "image,virtualmachine", "ssh-user=ubuntu")
```
Comment (guangbochen) [inline cmd/provider_cmd_harvester.go]: prefer not to default to `ubuntu`, we can just leave it as empty

Comment (futuretea) [reply cmd/provider_cmd_harvester.go]: This is `sampleFilters` show in help, but it is still wrong, I changed it to `virtualmachine=id1:id2:id3`

Comment (futuretea) [reply cmd/provider_cmd_harvester.go]: fixed.
eg:  only import image default/image-6skr8 and harvester-public/image-9lspw
```bash
terraformer-harvester import harvester -r image -f image=default/image-6skr8:harvester-public/image-9lspw
```

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**